README.md

```diff
@@ -32,6 +32,9 @@ let myClass = Css.style([
 - opacify
 - readable
 - transparentize
+- mix
+- shade
+- tint
  
```

Comment:
could you sort these to alphabetical order

Reply:
done